### PR TITLE
Jormun: durations of car and bike for distributed

### PR DESCRIPTION
**Pull request by mbounabi**
This PR will fix latest Artemis builds failures.

### CODE REVIEW

**Review by xlqian — APPROVED** [review]

File: source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py [hunk 1]
```diff
@@ -120,6 +121,9 @@ def _extend_journey(pt_journey, fallback_dp, fallback_extremity):
     pt_journey.duration += aligned_fallback.journeys[0].duration
     pt_journey.durations.total = pt_journey.duration
     pt_journey.durations.walking += aligned_fallback.journeys[0].durations.walking
+    pt_journey.durations.bike += aligned_fallback.journeys[0].durations.bike
+    pt_journey.durations.car += aligned_fallback.journeys[0].durations.car
+
```
Comment (kadhikari) [inline source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py:126]: What about crowfly? Is it also necessary in https://github.com/CanalTP/navitia/blob/43e3fd6a1e4247b311954125fdb5d059932097de/source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py#L53

Comment (mbounabi) [reply source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py:126]: In the specific case where the crowfly has a duration, it is associated to walking. Bike and car duration aren't involved.

**Review by kadhikari — APPROVED** [review]

**Review by pbougue — COMMENTED** [review]
Looks good, but you may check:
* the distance (should work pretty much the same as duration)
* if artemis sees this difference in output on distributed scenario

**Review by kinnou02 — APPROVED** [review]

**Comment by mbounabi** [comment]
@pbougue, the only case 'distances' is updated for a journey is for ridesharing for new_default scenario. Otherwise, it is never set and left to '0'. I think that another ticket is necessary to fill these fields.
I will however check if it doesn't cause any problem with Artemis.

**Comment by mbounabi** [comment]
Artemis reference needs to be updated with this PR:
https://github.com/CanalTP/artemis_references/pull/123

**Pull request merged**